File: KingdomCodeTool/wwwroot/Download/202502111449/Angular/shared/Sys_AppSettings.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { Sys_AppSettings } from './Sys_AppSettings.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class Sys_AppSettingsService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'Setting_Id', 'Code', 'Language_Id', 'Value', 'Description', 'Locked', 'SystemEditOnly', 'CreationDate', 'CreatedBy', 'LastUpdateDate', 'LastUpdatedBy'];

    List: Sys_AppSettings[] | undefined;
    ListFilter: Sys_AppSettings[] | undefined;
    FormData!: Sys_AppSettings;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "Sys_AppSettings";
    }
}

File: KingdomCodeTool/wwwroot/Download/202502111449/Service/Implement/Sys_AppSettingsService.cs
namespace Service_eHospital_DongNai_A_Config.Implement
{
    public class Sys_AppSettingsService : BaseService<Sys_AppSettings, ISys_AppSettingsRepository>
    , ISys_AppSettingsService
    {
    private readonly ISys_AppSettingsRepository _Sys_AppSettingsRepository;
    public Sys_AppSettingsService(ISys_AppSettingsRepository Sys_AppSettingsRepository) : base(Sys_AppSettingsRepository)
    {
    _Sys_AppSettingsRepository = Sys_AppSettingsRepository;
    }
    }
    }

